doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -124,11 +124,11 @@
 
 
 def plot_comparison(plot1, plot2, title1, title2):
-    fig, (ax1, ax2) = plt.subplots(ncols=2, figsize=(12, 6), sharex=True, sharey=True)
-    ax1.imshow(plot1, cmap='gray')
-    ax1.set_title(title1)
-    ax2.imshow(plot2, cmap='gray')
-    ax2.set_title(title2)
+    fig, ax = plt.subplots(ncols=2, figsize=(12, 6), sharex=True, sharey=True)
```

Comment:
@mkcor, I'm confused? In https://github.com/scikit-image/scikit-image/pull/7346#discussion_r1532120117 you indicate you wanted to revert these yourself?

Reply:
Right, I forgot to go through them! Sorry. I'll do it now if you're not currently editing the branch...

Reply:
Done ae5b047eb0557301.